solidity/dashboard/src/components/Timeline.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { useContext } from "react"
+
+const TimelineContext = React.createContext({
+  // May be useful in the future, eg. if we would like to be able to display the
+  // timeline events on alternating sides.
+  position: "left",
```

Comment:
Can we specify in the comment what are the values available here? eg. `left` and `right`? Maybe we could event create constants for this the for the `elementBreakpoint` position?

Reply:
Currently, we only support timeline on the `left` side. The `position` prop may be helpful in the future development of this component.